src/components/app/App.js

```diff
@@ -0,0 +1,73 @@
+import Header from '../header/Header'
+import Main from '../main/Main'
+import Home from '../home/Home'
+import Footer from '../footer/Footer'
+
+// const MOCK_DATA = [
```

Comment:
Would it perhaps be better for clarity to outsource the mock data to a data.json?

Reply:
I agree, either delete the comments or add the data to a `data.json` file.